### ISSUE TITLE
Windows Mixed Reality specific camera code has been moved into Windows Mixed Reality Camera Settings

### ISSUE THREAD

**Issue by elbuhofantasma** [microsoft/MixedRealityToolkit-Unity#8687]
## Describe the bug

When switching between Standalone and UWP in either order with any example scene open in the viewport and then changing the example scene, two 'Windows Mixed Reality specific camera code has been moved into Windows Mixed Reality Camera Settings' warnings are generated when the second example scene is opened.

## To reproduce

1. Import all MRTK .unitypackages in either the Standalone or UWP platform.
2. Open any example scene. (double click the .unity file for the example scene)
3. With the example scene open, switch to the opposite platform. (Either Standalone or UWP)
4. Once the platform switch is complete, open any different example scene.
    a. If the errors are not generated on the first try, switch back to the original platform and open a different example scene. Repeat until the warnings are generated.
5. Navigate to the Unity console and look at the warnings.

## Expected behavior

Switching between platforms will not effect the camera settings.

## Screenshots

![specificcamerawarning1](https://user-images.githubusercontent.com/65038391/94330316-5971a580-ff78-11ea-8885-2d7f33ddb84d.png)
![specificcamerawarning2](https://user-images.githubusercontent.com/65038391/94330317-5b3b6900-ff78-11ea-8629-b5865fd6772d.png)

## Your setup (please complete the following information)

- Unity Version 2019.4.10f1
- MRTK Version 20200925.7

## Target platform (please complete the following information)

- Standalone
- UWP

**Comment by IssueSyncBot** [comment]
We appreciate your feedback and thank you for reporting this issue.

Microsoft Mixed Reality Toolkit version 2 (MRTK2) is currently in limited support. This means that Microsoft is only fixing high priority security issues. Unfortunately, this issue does not meet the necessary priority and will be closed. If you strongly feel that this issue deserves more attention, please open a new issue and explain why it is important.

Microsoft recommends that all new HoloLens 2 Unity applications use MRTK3 instead of MRTK2.

Please note that MRTK3 was released in August 2023. It features an all new architecture for developing rich mixed reality experiences and has a minimum requirement of Unity 2021.3 LTS. For more information about MRTK3, please visit[https://www.mixedrealitytoolkit.org](https://www.mixedrealitytoolkit.org).

Thank you for your continued support of the Mixed Reality Toolkit!
